js/components/base-components/listItem.js

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var Icon = require('../icon');
+var classnames = require('classnames');
+
+var listItem = React.createClass({
+  handleSelect: function () {
+    this.props.handleSelect(this.props.id);
+  },
+
+  render: function () {
+    var classes = {
+      listItem: 'oo-list-item',
+      icon: 'icon',
```

Comment:
We should use `oo-` prefix for all classes. Generic class names can cause conflicts with the customer's CSS.